rosdep/python.yaml

```diff
@@ -5705,6 +5705,9 @@ python3-rafcon-pip:
   ubuntu:
     pip:
       packages: [rafcon]
+python3-requests:
+  ubuntu:
+    bionic: [python3-requests]
```

Comment:
@flixr Has a good point here; python3-requests is [available on Ubuntu](https://packages.ubuntu.com/search?keywords=python3-requests&searchon=names&suite=all&section=all) going all the way back to Xenial, so a wildcard would be best here.

Same goes for Debian, and Fedora.

Reply:
Xenial is the oldest supported platform so we don't need to do the wildcard syntax, just the straight 'ubuntu' rule without calling out the version.